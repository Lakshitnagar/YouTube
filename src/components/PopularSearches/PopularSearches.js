import React from 'react';
import './PopularSearches.scss';

const popularSearches = ["All Recommendations", "Sushant Singh Rajput", "Comedy", "Thrillers", "Spookiz", "TED Talks"];

function PopularSearches() {
    return (
        <div className="popularSearches">
            <div className="popularSearches__label">Popular Searches</div>

            {popularSearches.map((popularSearch)=>{
                return <div className="popularSearches__items">{popularSearch}</div>
            })}
        </div>
    );
}

export default PopularSearches;
