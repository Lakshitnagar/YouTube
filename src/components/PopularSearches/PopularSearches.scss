.popularSearches{
  display: flex;
  justify-content: flex-start;
  align-items: center;
  padding: 0 16px;
  height: 56px;
  box-shadow: 0 1px 0px 0px rgba(0, 0, 0, 0.14);

  .popularSearches__label{
    padding: 0 20px;
  }

  .popularSearches__items{
    margin-right: 25px;
  }
}
