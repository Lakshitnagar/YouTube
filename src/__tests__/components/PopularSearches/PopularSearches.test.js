import React from 'react';
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";
import PopularSearches from "../../../components/PopularSearches/PopularSearches";

describe('PopularSearches', function () {
    it('should render and match snapshot', function () {
        const component = <PopularSearches/>;

        const wrapper = shallow(component);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
