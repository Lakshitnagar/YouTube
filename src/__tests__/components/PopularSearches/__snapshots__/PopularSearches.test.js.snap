// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularSearches should render and match snapshot 1`] = `
<div
  className="popularSearches"
>
  <div
    className="popularSearches__label"
  >
    Popular Searches
  </div>
  <div
    className="popularSearches__items"
  >
    All Recommendations
  </div>
  <div
    className="popularSearches__items"
  >
    Sushant Singh Rajput
  </div>
  <div
    className="popularSearches__items"
  >
    Comedy
  </div>
  <div
    className="popularSearches__items"
  >
    Thrillers
  </div>
  <div
    className="popularSearches__items"
  >
    Spookiz
  </div>
  <div
    className="popularSearches__items"
  >
    TED Talks
  </div>
</div>
`;
